detections/endpoint/telegram_detected_access_suspicious_api_url.yml

```diff
@@ -0,0 +1,100 @@
+data_source:
+- Windows Security 4688
+name: Telegram Detected Access Suspicious API/URL
```

Comment:
Can we re arrange the yaml keys to match the key order as the other yamls. 

Eg- https://github.com/splunk/security_content/blob/develop/detections/endpoint/access_lsass_memory_for_dump_creation.yml